scrc/preprocessors/extractors/spider_specific/procedural_participation_extracting_functions.py

```diff
@@ -262,10 +311,190 @@ def get_regex():
+
+    ZH_Verwaltungsgericht_test_header = ['Verwaltungsgericht des Kantons Zürich 4. Abteilung', 'VB.2020.00452', 'Urteil', 'der 4. Kammer', 'vom 24. September 2020', 'Mitwirkend: Abteilungspräsidentin Tamara Nüssle (Vorsitz), Verwaltungsrichter Reto Häggi Furrer, Verwaltungsrichter Martin Bertschi, Gerichtsschreiber David Henseler.', 'In Sachen', 'A, vertreten durch RA B,', 'Beschwerdeführerin,', 'gegen', 'Migrationsamt des Kantons Zürich,', 'Beschwerdegegner,', 'betreffend vorzeitige Erteilung der Niederlassungsbewilligung,']
+
+    ZH_Sozialversicherungsgericht_test_header = ['Sozialversicherungsgerichtdes Kantons ZürichIV.2014.00602', 'II. Kammer', 'Sozialversicherungsrichter Mosimann, Vorsitzender', 'Sozialversicherungsrichterin Käch', 'Sozialversicherungsrichterin Sager', 'Gerichtsschreiberin Kudelski', 'Urteil vom 11. August 2015', 'in Sachen', 'X._', 'Beschwerdeführerin', 'vertreten durch Rechtsanwalt Dr. Kreso Glavas', 'Advokatur Glavas AG', 'Markusstrasse 10, 8006 Zürich', 'gegen', 'Sozialversicherungsanstalt des Kantons Zürich, IV-Stelle', 'Röntgenstrasse 17, Postfach, 8087 Zürich', 'Beschwerdegegnerin', 'weitere Verfahrensbeteiligte:', 'Personalvorsorgestiftung der Y._', 'Beigeladene']
+
 
+    ZG_Verwaltungsgericht_test_string = ' '.join(map(str, ZG_Verwaltungsgericht_test_header))
+    ZH_Steuerrekurs_test_string = ' '.join(map(str, ZH_Steuerrekurs_test_header))
+    ZH_Baurekurs_test_string = ' '.join(map(str, ZH_Baurekurs_test_header))
+    ZH_Obergericht_test_string = ' '.join(map(str, ZH_Obergericht_test_header))
+    ZH_Verwaltungsgericht_test_string = ' '.join(map(str, ZH_Verwaltungsgericht_test_header))
+    ZH_Sozialversicherungsgericht_test_string = ' '.join(map(str, ZH_Sozialversicherungsgericht_test_header))
 
+    namespace = {'language' : Language.DE}
 
+    zg_vg_json = ZG_Verwaltungsgericht(ZG_Verwaltungsgericht_test_string, namespace)
+    zg_vg = json.loads(zg_vg_json)
+    # print(zg_vg)
+    assert zg_vg['plaintiffs'][0]['legal_counsel'][0]['name'] == 'B._ AG'
+    assert zg_vg['plaintiffs'][0]['legal_counsel'][0]['legal_type'] == 'legal entity'
+    # zh_sr = ZH_Steuerrekurs(ZH_Steuerrekurs_test_string, namespace)
+    # zh_br = ZH_Baurekurs(ZH_Baurekurs_test_string, namespace) 
+    # zh_og = ZH_Obergericht(ZH_Obergericht_test_string, namespace)
+    # zh_vg= ZH_Verwaltungsgericht(ZH_Verwaltungsgericht_test_string, namespace)
+    # zh_svg = ZH_Sozialversicherungsgericht(ZH_Sozialversicherungsgericht_test_string, namespace)
```

Comment:
Probably not my business, but could we maybe put this in a separate test module? Would love to extend on it down the line for other spiders. And if we put it separately we can call the test procedure as a part of the pipeline instead of just for the debugging 😉

Reply:
Yes, I think this is a good idea.

Reply:
Alright. Should I create a testing module in the utils directory? Or somewhere else?

Reply:
Yes, I think the utils directory would be a good place. Please also talk to Alex, I think he proposed this in the first place, and he is working on similar things, as far as I know. Also, Christian is involved in this area I think. I guess it would be best if you worked on this together :)